es-ds-docs/pages/molecules/skeleton.vue

```diff
@@ -0,0 +1,164 @@
+    loading.value = false;
+};
+
+const esSkeletonProps = [
+    ['animation', 'String', 'wave', `Options are 'wave', 'fade', or 'none'`],
+    ['height', 'String', '1rem', `Manually set height`],
+    ['width', 'String', 'auto', `Manually set width`],
+    ['size', 'String', 'null', `Manually set width and height, overriding 'height' and 'width' props`],
+];
+
+const esSkeletonWrapperProps = [
+    ['loading', 'Boolean', 'false', `Determines whether to show loading template or default content`],
+];
+
+const esSkeletonImgProps = [
+    ['aspect', 'String', '16:9', `Adds a container around the skeleton image with given aspect ratio`],
+    ['noAspect', 'Boolean', 'false', `Override the default aspect ratio and height, width, or size`],
+    ['...EsSkeleton props', '...', '...', `Takes all 'esSkeleton' props shown above`],
+];
+
+const { $prism } = useNuxtApp();
+const compCode = ref('');
+const docCode = ref('');
+onMounted(async () => {
+    if ($prism) {
+        const compSource = await import('@energysage/es-ds-components/components/es-skeleton.vue?raw');
+        // eslint-disable-next-line import/no-self-import
+        const docSource = await import('./skeleton.vue?raw');
+        compCode.value = $prism.normalizeCode(compSource.default);
+        docCode.value = $prism.normalizeCode(docSource.default);
+        $prism.highlight();
+    }
+
+    startLoading();
+});
+</script>
+
+<template>
+    <div>
+        <h1>Skeleton</h1>
+        <p>
+            Extended from
+            <nuxt-link
+                href="https://v3.primevue.org/skeleton/"
+                target="_blank">
+                PrimeVue Skeleton
+            </nuxt-link>
+            and
+            <nuxt-link
+                href="https://bootstrap-vue.org/docs/components/skeleton/"
+                target="_blank">
+                Bootstrap Skeleton
+            </nuxt-link>
+            styles
+        </p>
+
+        <h2>Basic examples</h2>
+        <p>Wave Animation:</p>
+        <es-row class="mb-200">
+            <es-col lg="6">
+                <es-skeleton />
+                <es-skeleton width="75%" />
+                <es-skeleton height="3rem" />
+            </es-col>
+            <es-col
+                lg="6"
+                class="d-flex">
+                <es-skeleton size="5rem" />
+                <es-skeleton
+                    size="4rem"
+                    class="rounded-circle ml-100" />
+            </es-col>
+        </es-row>
+
+        <p>Fade Animation:</p>
+        <es-row class="mb-500">
+            <es-col lg="6">
+                <es-skeleton
+                    animation="fade"
+                    height="33px"
+                    width="100%" />
+                <es-skeleton
+                    animation="fade"
+                    height="25px"
+                    width="100%" />
+            </es-col>
+        </es-row>
+
+        <h2>Helper components</h2>
+
+        <h3>Skeleton Wrapper</h3>
```

Comment:
Nit: sentence case to match our content guidelines on headings/labels - "Skeleton wrapper"